study-builder/studies/osteo/study.conf

```diff
@@ -166,8 +166,24 @@
       ]
     }
   ],
-
   "activities": [
+    {
+      "filepath": ${FamilyHistoryPath}"family-history-top-level.conf",
```

Comment:
Adding family history initially to Osteo.
Up to now, every activity configuration file has belonged to a single study (except for snippets).
Trying to make the family history configuration shareable  between all the CMI studies.

Reply:
This is nitpicky, but I find it easier to read if the path variable itself doesn't include a slash, and we add in the slash we use use it, like this:
```
${FamilyHistoryPath}/family-history-about-you.conf
${FamilyHistoryPath}/family-history-aunt-uncle.conf
```

Reply:
I share your dislike for trailing / in variables. In this very particular case I had wanted to keep my options open to possibility that all the family history would have been in same folder as all the osteo configs, in which case FamilyHistoryPath would have been blank. We are not doing that so:
c0586eec

Reply:
I have also gone on to refactor the HOCON further so that the activity entry for family history is `include required`.
f49f200bf971a706e23f4604d35c7c49d1be219b

Needed to tweak resolution substitution variables for study.conf file in Java code:
a4e7789ca200e378659b85a0bf55ea655dd27bcb